crypto/LPdir_win.c

```diff
@@ -105,6 +106,7 @@ const char *LP_find_file(LP_DIR_CTX **ctx, const char *directory)
                 errno = ENOMEM;
                 return 0;
             }
+            memset(wdir, 0, amount);
```

Comment:
This isn't really needed.

Reply:
I didn't think so, but the original code used ```calloc``` so I kept the semantics.

Reply:
updated commit pushed to remove it; please re-review.